ntbs-service/Services/AlertService.cs

```diff
@@ -40,6 +48,31 @@ public async Task DismissAlertAsync(int alertId, string userId)
         public async Task<bool> AddUniqueAlertAsync(Alert alert)
         {
             var matchingAlert = await _alertRepository.GetAlertByNotificationIdAndTypeAsync(alert.NotificationId, alert.AlertType);
+            if ((matchingAlert != null && (matchingAlert != null ? matchingAlert?.AlertType != AlertType.TransferRequest : true)) 
+                || (matchingAlert.AlertType == AlertType.TransferRequest && matchingAlert.AlertStatus == AlertStatus.Open))

```

Comment:
I think this can be simplified (correct me if I'm wrong):
```
if (matchingAlert != null
    || (matchingAlert.AlertType == AlertType.TransferRequest && matchingAlert.AlertStatus == AlertStatus.Open))
```

Reply:
We spoke and you said this is probably redudant code now and the un-unique is handled otuside of here.
You said you wanted to add some tests - I think some unit tests on this stuff might be sensible (assuming it doesn't already have decent coverage).